code/core/src/core-server/presets/common-preset.ts

```diff
@@ -104,8 +104,32 @@ export const favicon = async (
 
 export const babel = async (_: unknown, options: Options) => {
   const { presets } = options;
-
-  return presets.apply('babelDefault', {}, options);
+  const babelDefault = await presets.apply('babelDefault', {}, options);
+  return {
+    ...babelDefault,
+    // This override makes sure that we will never transpile babel further down then the browsers that storybook supports.
+    // This is needed to support the mount property of the context described here:
+    // https://storybook.js.org/docs/writing-tests/interaction-testing#run-code-before-each-test
+    overrides: [
+      {
```

Comment:
Don't you need to spread in babelDefault.overrides here?

Reply:
good point